app/views/exercises/show.html.slim

```diff
@@ -7,10 +7,19 @@ h1
   - if policy(@exercise).edit?
     = render('shared/edit_button', object: @exercise)
 
+
 = row(label: 'exercise.title', value: @exercise.title)
 = row(label: 'exercise.user', value: link_to_if(policy(@exercise.author).show?, @exercise.author, @exercise.author))
 = row(label: 'exercise.description', value: render_markdown(@exercise.description))
-= row(label: 'exercise.execution_environment', value: link_to_if(policy(@exercise.execution_environment).show?, @exercise.execution_environment, @exercise.execution_environment))
+= row(label: 'exercise.execution_environment') do
+  = link_to_if(policy(@exercise.execution_environment).show?, @exercise.execution_environment, @exercise.execution_environment)
+  - if @exercise.default_execution_environment
+    .default-execution-environment-warning
```

Comment:
default execution environment warning? Is that specifically for imported exercises?